guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -211,6 +211,36 @@ To configure network services on {Project}'s integrated {SmartProxy}, complete t
 # hammer proxy info --name "_{foreman-example-com}_"
 ----
 
+[[Configuring_Networking-Multiple_DHCP_Subnets_via_Puppet]]
+==== Multiple DHCP Subnets via Puppet
+
+The installer have options available only for a single DHCP subnet. One way to define more than one subnet is via Puppet, the tool that the installer utilizes to perform the configuration.
```

Comment:
```suggestion
The {foreman-installer} options allow only for a single DHCP subnet. One way to define more than one subnet is by using Puppet.
```

Reply:
I'm not sure we need to mention that Puppet is in the installer, since this will change and also since it has no real relevance to using Puppet to add more subnets.

Reply:
Indeed. Puppet is an implementation detail of the installer.

Reply:
I can remove the reference here.